New File send_email.py added

harshalkh opened the pull request:
New file for sending email

2Max4 commented on send_email.py:
```diff
@@ -0,0 +1,37 @@
+
+sender_from_email = "sender@gmail.com"
+receiver_to_email = "receiver@gmail.com"
+password = input("Type your password and hit enter:")
```
Could you please change the input to getpasswd - I think this might be better because you cannot see the direkt input.

harshalkh replied:
yes changed the implementation to getpass()

2Max4 commented on send_email.py:
```diff
@@ -0,0 +1,37 @@
+message["From"] = sender_from_email
+message["To"] = receiver_to_email
+
+# Create the HTML version of email content message
```
Could you please include the files that are generated when running the network_test_class.py or the Analytics.ipynb. They are located in webpage/figures (but only if analytics has been run)

harshalkh replied:
which type of files are generated?? And do we need them as attachments??

Skupino replied:
If you run the analytics after network_test performed for a while, 3 diagrams are generated.

harshalkh replied:
I have added a logic to attach all the reports in email. please have a look.

2Max4 commented on send_email.py:
```diff
@@ -0,0 +1,37 @@
+
+message = MIMEMultipart("alternative")
+message["Subject"] = "Network Analysis Report"
+message["From"] = sender_from_email
```
It would be great if the user would be promted to enter his (google) email information as well as the reciever of the report

harshalkh commented:
> Hey Harshalkh, thanks for your help!
> It would be aswesome if you could check the comments and implement them to your code!
> 
> It would also be cool if you could write a class which we can then call from a main Module to send the email.
> 
> Greetings

Hello, I have implemented the changes as per review comments. Also added class structure for SendEmail. Please review.

2Max4 commented:
Hey @harshalkh revied your changes - for me that looks absolutely great! I'll merge it - if there is anything to change I will mark it in Issue #17 . Thanks for your great support!
